security-analysis/src/main/java/com/powsybl/security/json/SecurityAnalysisParametersJsonModule.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2018, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.json;
+
+import com.powsybl.loadflow.json.LoadFlowParametersJsonModule;
+import com.powsybl.security.SecurityAnalysisParameters;
+
+/**
+ * @author Sylvain Leclerc <sylvain.leclerc@rte-france.com>
+ */
+public class SecurityAnalysisParametersJsonModule extends LoadFlowParametersJsonModule {
```

Comment:
As there is no inheritence between LoadFlowParameters and SecurityAnalysisParamaters, I'm not sure it's a good idea that SecurityAnalysisParametersJsonModule inherit from LoadFlowParameters. It should be a composition.